### PR TITLE
Joel upgrade docs

### PR DESCRIPTION
**Pull request by joeldudleyr3**

### CODE REVIEW

File: docs/source/node-database-access-h2.rst [hunk 1]
```diff
@@ -44,5 +60,41 @@ The node requires a database password to be set when the database is exposed on
...
+interface for you to query them using SQL.
+
+.. _h2_relative_path:
+
+Connecting directly to the node's ``persistence.mv.db`` file
+------------------------------------------------------------
+
+You can also use the H2 Console to connect directly to the node's ``persistence.mv.db`` file:
+
+    ``jdbc:h2:~/path/to/file/persistence``
```
Comment (szymonsztuka) [inline docs/source/node-database-access-h2.rst]: Ensure the node is off, as accessing to the database file  requires exclusive access, if not H2 will return error  "Database may be already in use: null. Possible solutions: close all other connection(s); use the server mode [90020-196]"

File: docs/source/node-database-access-h2.rst [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+-------------------------------------
+
+The database (a file called ``persistence.mv.db``) is created when the node first starts up. By default, it has an
+administrator user ``sa`` and a blank password. The node requires the user with administrator permissions in order to
+creates tables upon the first startup or after deploying new CorDapps with their own tables. The database password is
+required only when the H2 database is exposed on non-localhost address (which is disabled by default).
+
+This username and password can be changed in node configuration:
+
+ .. sourcecode:: groovy
```
Comment (szymonsztuka) [inline docs/source/node-database-access-h2.rst]: New line

Comment (joeldudleyr3) [reply docs/source/node-database-access-h2.rst]: What do you mean, sorry?

Comment (szymonsztuka) [reply docs/source/node-database-access-h2.rst]: Just add new line, otherwise the code doesn't render.

Comment (joeldudleyr3) [reply docs/source/node-database-access-h2.rst]: Thanks :)

**Pull request merged**